src/components/field-label/field-label.component.tsx

```diff
@@ -42,7 +41,7 @@ const FieldLabel: React.FC<FieldLabelProps> = ({ field, customLabel }) => {
         )}
         {hasTooltip && (
           <Information
-            size={20}
+            size={16}
```

Comment:
After this change:

![CleanShot 2025-06-09 at 16 34 26@2x](https://github.com/user-attachments/assets/8771b01a-5221-4972-bfa8-96993b8ae174)